cmd/kogito/command/context/root.go

```diff
@@ -71,6 +71,7 @@ func (i *rootCommand) RegisterHook() {
 func (i *rootCommand) InitHook() {
 	i.flags = rootCommandFlags{}
 	i.command.PersistentFlags().StringVar(&i.flags.cfgFile, "config", "", "config file (default is $HOME/.kogito/config.yaml)")
+	i.command.PersistentFlags().StringVarP(&outputFormat, "output", "o", "", "output format")
```

Comment:
We should display the supported format.

Reply:
Changed to:
- o, -- output string   output format (supported value is 'json')